AnkiDroid/src/main/java/com/ichi2/anki/AbstractFlashcardViewer.java

```diff
@@ -1945,8 +1966,19 @@ protected void displayCardQuestion() {
         updateCard(displayString);
         hideEaseButtons();
 
+        // Check if it should use the general 'Timeout settings' or the ones specific to this deck
+        if (mOptUseGeneralTimerSettings){
```

Comment:
down to just whitespace things now, not sure why this did not trigger our codacy rules - I'll check that separately 
```suggestion
        if (mOptUseGeneralTimerSettings) {
```